dask/bag/core.py

```diff
@@ -95,7 +95,7 @@ def optimize(dsk, keys, **kwargs):
 
 
 def to_textfiles(b, path, name_function=str, compression='infer',
-                 encoding=system_encoding):
+                 encoding=system_encoding, **kwargs):
```

Comment:
Can I suggest a `compute=True` keyword argument that reverts back to the old behavior if False?  This can be useful sometimes.

```python
result = Bag(...)
if compute:
    result.compute()
else:
    return result
```